nixos/modules/services/web-servers/caddy.nix

```diff
@@ -6,90 +6,101 @@ let
   cfg = config.services.caddy;
   configFile = pkgs.writeText "Caddyfile" cfg.config;
 in
-{
-  options.services.caddy = {
-    enable = mkEnableOption "Caddy web server";
+  {
```

Comment:
Thank you for this. We are in nit-picking territory but the indentation is incorrect. It should either be:

```
in
{
  options....
```
or
```
in {
  options...
```
That basically throws off the indentation for the rest of the PR.